### ISSUE TITLE
[Feature][Connector-V2] Need help for supporting multi-table sink feature

### ISSUE THREAD

**Issue by Hisoka-X** [apache/seatunnel#5652]
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/seatunnel/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

## Purpose

After #5395 finnished. SeaTunnel sink can implement multi-table interface, with this feature. We can aovid shuffle in zeta. We should add implement multi-table interface for all SeaTunnel sink, also need test case. Please refer Jdbc from https://github.com/apache/seatunnel/pull/5620. 

## Please pick connector you want to implement it!

| Type | Connector | Contributer | Status | PR |
| ---- | ---- | ---- | ---- | ---- |
| Sink | Console | | Done | | 
| Sink | Assert | | Done | |  
| Sink | Kafka | @misi1987107  | | |  
| Sink | Http | @jackyyyyyssss | | | 
| Sink | Feishu | | | |  
| Sink | Hive | @FlechazoW | | |   
| Sink | Clickhouse | @mrtisttt |Done | | 
| Sink | ClickhouseFile | | | |  
| Sink | Jdbc | | Done | |  
| Sink | Kudu |@Carl-Zhou-CN | | | 
| Sink | EmailSink | @liugddx |Done | |  
| Sink | HdfsFile | @dengd1937 | | |  
| Sink | LocalFile | @ruanwenjun | Done | #5931 |  
| Sink | OssFile | @liugddx| Done| |  
| Sink | OssJindoFile | @liugddx|Done | |  
| Sink | CosFile | |Done | |   
| Sink | DingTalk | | | |  
| Sink | Elasticsearch | @CosmosNi | | |  
| Sink | IoTDB | | | |  
| Sink | Neo4j | | | |  
| Sink | FtpFile | | Done| |  
| Sink | SftpFile | |Done | | 
| Sink | Socket | | | |  
| Sink | Redis |@jackyyyyyssss | | | 
| Sink | DataHub |@jia17| Done| | 
| Sink | Sentry | | | |  
| Sink | MongoDB | @chl-wxp | Done | |  
| Sink | S3File | | Done| |  
| Sink | AmazonDynamodb | | | |  
| Sink | Cassandra | | | | 
| Sink | StarRocks | | Done| |  
| Sink | InfluxDB |@531651225 | | |  
| Sink | GoogleFirestore |@lvshaokang  | | | 
| Sink | Tablestore | | | |   
| Sink | Slack | | | |  
| Sink | RabbitMQ |@kulabs8 | | |  
| Sink | Doris | @gnehil | | |  
| Sink | Maxcompute | | | |   
| Sink | S3Redshift | | | |  
| Sink | TDengine |@jia17 |Done | |  
| Sink | SelectDBCloud |  |  | | 
| Sink | Hbase | @BruceWong96 | | |   
| Sink | Rocketmq | | | |  
| Sink | Paimon | @dailai | | |

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by gnehil** [comment]
Please assign Doris sink task to me, thx

**Comment by FlechazoW** [comment]
paimon\hive\cassandra please assign to me.

**Comment by Carl-Zhou-CN** [comment]
Please assign me kudu

**Comment by Hisoka-X** [comment]
> paimon\hive\cassandra please assign to me.

Hi @FlechazoW , I just assign hive to you. I will assgin other to you after you create a PR for hive.

**Comment by hailin0** [comment]
> Please assign me kudu

@Carl-Zhou-CN 
Please create  issue link to this comment

**Comment by Carl-Zhou-CN** [comment]
> > Please assign me kudu
> 
> @Carl-Zhou-CN Please create issue link to this comment

done

**Comment by dengd1937** [comment]
Please HdfsFile sink task to me, thx

**Comment by Carl-Zhou-CN** [comment]
Please Clickhouse sink task to me, thx

**Comment by lightzhao** [comment]
kafka and pulsar also require support multi-table sink feature? What is the meaning of this?

**Comment by cyd257666** [comment]
Please assign starrocks sink task to me, thx

**Comment by shuihua777** [comment]
@hailin0 Please Starrocks sink task to me, thx

**Comment by lvshaokang** [comment]
@hailin0  Plz assign me GoogleFirestore sink task, thx

**Comment by 531651225** [comment]
@hailin0 InfluxDB sink task to me, thx

**Comment by jackyyyyyssss** [comment]
@Hisoka-X please assign me redis sink thx

**Comment by jackyyyyyssss** [comment]
@Hisoka-X please assign me http sink thx

**Comment by UFOXD** [comment]
waitting for kafka sink

**Comment by CosmosNi** [comment]
@Hisoka-X please assign me Elasticsearch sink thx

**Comment by BruceWong96** [comment]
@Hisoka-X Please assign me Hbase sink thx.

**Comment by skyoct** [comment]
@Hisoka-X I would like to try DingTalk

**Comment by corgy-w** [comment]
@Hisoka-X  Why is it that when I go to implement mail's multi-sink and just use from multi-interface, exec will throw `Action name is duplicated: [pipeline-1 [Sink[0]-EmailSink]]` ，i didn't post config because it's usual changes.

**Comment by Hisoka-X** [comment]
> @Hisoka-X Why is it that when I go to implement mail's multi-sink and just use from multi-interface, exec will throw `Action name is duplicated: [pipeline-1 [Sink[0]-EmailSink]]` ，i didn't post config because it's usual changes.

Please create an issue for this.

**Comment by corgy-w** [comment]
> Please create an issue for this.

Ok,Please assign me this case and I'll be monitoring it closely.

**Comment by corgy-w** [comment]
> Ok,Please assign me this case and I'll be monitoring it closely.

@Hisoka-X #7368

**Comment by akulabs8** [comment]
Hey, I am taking the Slack one for starters @davidzollo  @Hisoka-X 

Just one question, we have to do both the sink and source of a particular connector simultaneously ?

**Comment by akulabs8** [comment]
https://github.com/apache/seatunnel/pull/8994

Added this for RabbitMq

**Comment by davidzollo** [comment]
> Hey, I am taking the Slack one for starters [@davidzollo](https://github.com/davidzollo) [@Hisoka-X](https://github.com/Hisoka-X)
> 
> Just one question, we have to do both the sink and source of a particular connector simultaneously ?

One of Source and Sink will be ok

**Comment by davidzollo** [comment]
If there is anybody who'd like to implement these connector features, please leave a related message like `I'd like to implement XX sink` and then you can submit your PR within four weeks. 

A very kindly reminder: Other guys can pick it up if the related PR is not submitted within four weeks. ^-^

Here are some helpful resources to get started:  
- [Contribution Setup Guide](https://seatunnel.apache.org/docs/contribution/setup)  
- [Code Submission Guide](https://seatunnel.apache.org/community/submit_guide/submit-code)  

By the way, since this might be your first contribution, I think we can have a more in-depth communication to help you get familiar with SeaTunnel. Feel free to contact me on LinkedIn ([David Zollo](https://www.linkedin.com/in/davidzollo)) or WeChat ID(davidzollo). When adding me, please let me know your GitHub ID. I'd be happy to assist you!

**Comment by mrtisttt** [comment]
@davidzollo Hi, It seems that the multi-table sink feature of ClickHouse has not been implemented? I'm interested in it, but seeing that someone has claimed it, can it be assigned to me?

**Comment by davidzollo** [comment]
> [@davidzollo](https://github.com/davidzollo) Hi, It seems that the multi-table sink feature of ClickHouse has not been implemented? I'm interested in it, but seeing that someone has claimed it, can it be assigned to me?

Sorry for reply so late.  Thanks for the PR.
There are four weeks to implement it if there're someone has claimed it. If no PR is provided within four weeks, anybody can leave a comment under this issue to claim it again!

**Comment by davidzollo** [comment]
I have updated this table. Please leave a message if anybody would like to implement the multi - table sink feature for any connector.

**Comment by davidzollo** [comment]
Please leave a message if anybody would like to implement the multi - table sink feature for any connector.

**Comment by chl-wxp** [comment]
MongoDB done

**Comment by misi1987107** [comment]
Please assign Kafka sink task to me, thx

**Comment by davidzollo** [comment]
This issue is too old, I've created a new issue for tracking the undo connectors [[Feature][Connector] Implement multi-table sink support for remaining connectors #10426](https://github.com/apache/seatunnel/issues/10426)
